packages/visualizations-react/package.json

```diff
@@ -47,31 +53,60 @@
     },
     "dependencies": {
         "@opendatasoft/visualizations": "^0.9.0",
+        "@testing-library/jest-dom": "^5.11.9",
+        "@testing-library/react": "^11.2.5",
+        "@types/jest": "^29.0.0",
```

Comment:
I would have expected these 3 libs should be in `devDependencies`, not in `dependencies`, as they should absolutely not be packaged in regular builds. Is there a specific reason?

Reply:
No it's a merge problem, they've been there back and forth with eslint 👌